### DISCUSSION TITLE
Limit completion to `Field` when completing inside a struct

### DISCUSSION THREAD

**Discussion by njkevlani** [hrsh7th/nvim-cmp#982] [Q&A]
Is it possible to limit completion to `Field` when completing inside a struct?

## Example
Only `a` and `b` should be shown in the completion options.
![Screenshot_2022-05-10_06-54-30](https://user-images.githubusercontent.com/15014949/167524851-9abb2f32-c137-4e46-a8cc-b0b9a07d5310.png)

### Example Code
```go
package main

import "fmt"

type testStruct struct {
	a int
	b int
}

func main() {
	s := testStruct{a: 10}
	fmt.Println(s)
}
```
### Configs
- [gopls config](https://github.com/njkevlani/dotfiles/blob/9dba7ab/nvim-lsp/.config/nvim/lua/gopls-lsp.lua)
- [nvim-cmp config](https://github.com/njkevlani/dotfiles/blob/9dba7ab/nvim-lsp/.config/nvim/lua/nvim-cmp.lua)

**Comment by Shougo** [comment]
It seems LSP's problem instead of nvim-cmp.
nvim-cmp cannot detect  the syntax.
